src/cmd/linuxkit/run_qemu.go

```diff
@@ -67,8 +67,12 @@ func init() {
 		defaultArch = "aarch64"
 	case "amd64":
 		defaultArch = "x86_64"
+	case "s390x":
+		defaultArch = "s390x"
 	}
 	switch {
+	case runtime.GOARCH == "s390x":
+		defaultAccel = "kvm"
```

Comment:
Does the `hackKVM()` case not work for s390x? That case handles fallback to tcg if for some reason kvm is not usable by the current user.

Reply:
on s390x, TCG does not provide good machine support, so virtualization with KVM is essentially the only way to execute images on s390x

Reply:
Is https://wiki.qemu.org/Documentation/Platforms/S390X out of date then, it seems to suggest tcg works for modern distros with current (2.12+) versions of QEMU.

It might not be the best available option, and we expect that, but when it is the only option then `linuxkit run` tool (which is a dev not production tool) usually allows it (e.g. for existing architectures it does).

Reply:
I too would prefer if I could run s390x images on another architecture, no matter how slow...
Or is this a matter of virtio/emulated HW support not being avail?

Reply:
We only work and test KVM, but it might happen to work by chance on other platforms. Therefore, in the PR for s390 only kvm is enabled.

Reply:
I would still prefer if we don't actively disable it for now, so that we can at least easily test. If we find that it doesn't work at all, we can disable it later

Reply:
I agree and I think this just means removing the various s390x special cases currently added here around `accel` and `kvm` vs `kvm:tcg`. I'd say there's no obligation for @alicefr to test the result in a non-kvm environment though.

Reply:
ok, would it be reasonable to set as default accelerator `kvm`, and then overwrite it with `-accel`?

Reply:
@alicefr yes, i think that would be fine.

Reply:
Great!:)